cpp/ql/test/query-tests/Likely Bugs/Leap Year/UncheckedLeapYearAfterYearModification/test.cpp

```diff
@@ -680,3 +680,57 @@ void mkDateTest(int year)
 	}
 	// ...
 }
+
+void unmodified1()
+{
+	SYSTEMTIME st;
+	FILETIME ft;
+	WORD w;
+
+	GetSystemTime(&st);
+
+	w = st.wYear;
+
+	SystemTimeToFileTime(&st, &ft); // GOOD - no modification
+}
+
+void unmodified2()
+{
+	SYSTEMTIME st;
+	FILETIME ft;
+	WORD *w_ptr;
+
+	GetSystemTime(&st);
+
+	w_ptr = &(st.wYear);
+
+	SystemTimeToFileTime(&st, &ft); // GOOD - no modification
+}
+
+void modified3()
+{
+	SYSTEMTIME st;
+	FILETIME ft;
+	WORD *w_ptr;
+
+	GetSystemTime(&st);
+
+	st.wYear = st.wYear + 1;
+
+	SystemTimeToFileTime(&st, &ft); // BAD
+}
+
+void modified4()
+{
+	SYSTEMTIME st;
+	FILETIME ft;
+	WORD *w_ptr;
+
+	GetSystemTime(&st);
+
+	st.wYear++;
+	st.wYear++;
+	st.wYear++;
```

Comment:
These three lines are `BAD` according to `UncheckedLeapYearAfterYearModification.expected`. Please annotate these lines as well as the three lines earlier in this file that are relevant to this query.

Reply:
Done.